client/my-sites/email/form/mailboxes/components/form.tsx

```diff
@@ -0,0 +1,119 @@
+import { Button } from '@automattic/components';
+import { translate, useRtl } from 'i18n-calypso';
+import { useState } from 'react';
+import { MailboxForm } from 'calypso/my-sites/email/form/mailboxes';
+import { MailboxField } from 'calypso/my-sites/email/form/mailboxes/components/field';
+import {
+	EmailProvider,
+	FormFieldNames,
+	GoogleMailboxFormFields,
+	MailboxFormFieldBase,
+	TitanMailboxFormFields,
+} from 'calypso/my-sites/email/form/mailboxes/types';
+
+interface MailboxFormWrapperProps {
+	domains?: string[];
```

Comment:
```suggestion
	domains?: string[] | string;
```

Reply:
Removing this for now because it's not implemented yet